tasks/e2e-kitchensink.sh

```diff
@@ -153,11 +163,11 @@ npm link $root_path/packages/eslint-config-react-app
 npm link $root_path/packages/react-dev-utils
 npm link $root_path/packages/react-scripts
 
-# ...and we need  to remove template's .babelrc
+# ...and we need to remove template's .babelrc
 rm .babelrc
 
 # Test the build
-NODE_PATH=src REACT_APP_SHELL_ENV_MESSAGE=fromtheshell npm run build
+NODE_PATH=src NODE_ENV=production REACT_APP_SHELL_ENV_MESSAGE=fromtheshell npm run build
```

Comment:
Our babel preset requires it to be set so I simply set it on all calls.

I went ahead and removed it from the `build` calls.

We still need to specify them on the mocha calls since we switched from `latest` to `react-app` and it's [`react-app`] executed by mocha via `babel-register` (which apparently doesn't auto-set env to `test`).